web/src/components/application-form/index.tsx

```diff
@@ -66,7 +66,12 @@ const createCloudProviderListFromPiped = ({
     return [{ name: "None", value: "" }];
   }
 
-  return piped.cloudProvidersList
+  const providerList: Array<{ name: string; type: string }> = [
+    ...piped.cloudProvidersList,
+    ...piped.platformProvidersList,
+  ];
```

Comment:
Should we ensure no duplication?

Reply:
Nice point 👍 But I just notice we don't have rules to disallow duplicated XXXProvider names currently, isn't it? In such case, explicitly making the names duplicately display on UI will aware users of their configuration, I think 🤔

Reply:
No, I didn't mean to avoid duplication inside the platform provider list. Instead of that, we're now merging 2 lists: `cloudProviders` and `platformProviders`, so is there any case where both of them are storing the same list?

Reply:
For the new piped, CloudProvider configuration list will be removed before report meta (ref: [here](https://github.com/pipe-cd/pipecd/blob/master/pkg/config/piped.go#L97-L98)) and for the old one, the piped configuration contains only CloudProvider, so I don't think PlatformProvider and CloudProvider list can be exactly same (except users manually copy all value in CloudProvider to PlatformProvider and keep both lists)

Reply:
Got it. Just want to confirm that. 👍